debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -904,6 +915,12 @@ protected String slotName() {
     }
 
     protected boolean dropSlotOnStop() {
+        if (getConfig().hasKey(DROP_SLOT_ON_STOP.name())) {
+            return getConfig().getBoolean(DROP_SLOT_ON_STOP);
+        }
+        if (getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name())) {
+            return getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name());
```

Comment:
hasKey() -> getBoolean()

Reply:
@ Seems that copy and paste is more prone to bugs :-(